libs/flask_restx/api.py

```diff
@@ -873,9 +873,13 @@ def _deferred_blueprint_init(self, setup_state):
             setup_state.add_url_rule = MethodType(
                 Api._blueprint_setup_add_url_rule_patch, setup_state
             )
-        if not setup_state.first_registration:
-            raise ValueError("flask-restx blueprints can only be registered once.")
-        self._init_app(setup_state.app)
+            
+        # https://github.com/flask-restful/flask-restful/issues/509
+        # if not setup_state.first_registration:
+        #     raise ValueError("flask-restx blueprints can only be registered once.")
+        # self._init_app(setup_state.app)
+        if setup_state.first_registration:
+           self._init_app(setup_state.app)
```

Comment:
I had to make this change to a lib to allow creating an alias,

@morpheus65535 is this a file that can be modified or is it clobbered? 

If its clobbered any idea how we can make an alias of the api under `/api/v1` ?